crates/bevy_app/src/plugin.rs

```diff
@@ -90,6 +150,35 @@ pub trait Plugin: Downcast + Any + Send + Sync {
     fn is_unique(&self) -> bool {
         true
     }
+
+    /// Updates the plugin to a desired [`PluginState`].
```

Comment:
I think that renaming this to something like "process" and then describing how this performs the associated processing step would be clearer :)

Reply:
Done